azerothcore/azerothcore-wotlk
# Removed hardcoded emotes from Ahune and moved them to DB

## Changes Proposed:
-  Added missing DB entries for Ahune (25740) to creature_text
-  Removed hard-coded text from boss_ahune.cpp

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4146

## SOURCE:


## Tests Performed:

- Tested in-game
- OS: Win10

## How to Test the Changes:


1. Enter Ahune event
2. Activate encounter and check that emotes occur

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
